recipes/exiv2/all/conanfile.py

```diff
@@ -48,6 +48,10 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+        if self.options.with_xmp == "bundled":
+            # recipe has bundled xmp-toolkit-sdk of old version
+            # avoid conflict with a future xmp recipe
+            self.provides = "xmp-toolkit-sdk"
```

Comment:
`self.provides` is a string iterable of strings [docs](https://docs.conan.io/en/latest/reference/conanfile/attributes.html#provides)
To be safe, I would initialize it to a list and then append in `configure`.
Just to be sure...

Reply:
With https://github.com/conan-io/conan-center-index/pull/6633/files#r681226625
```suggestion
            self.provides.append("xmp-toolkit-sdk")
```

Reply:
Docs says it is a string with a recipe name (The attribute value should be a string with a recipe name or a tuple of such recipe names.). List does not satisfy this requirement so I assigned a string.

Reply:
I suggested a list + append to avoid overwriting `self.provides` when/if it would be added.